region/crateria/central/Landing Site.json

```diff
@@ -785,6 +785,23 @@
       ],
       "devNote": "Unlocking doors would be done ahead-of-time when the door is opened, i.e. when obstacle 'C' is set."
     },
+    {
+      "link": [3, 1],
+      "name": "Use Flash Suit, Horizontal Spark",
+      "requires": [
+        "canHorizontalShinespark",
+        {"useFlashSuit": {}},
+        {"or": [
+          {"shinespark": {"frames": 95, "excessFrames": 34}},
+          {"and": [
+            "HiJump",
```

Comment:
This strat could use more explanation.  To be different from [4,1] you would jump from the top ledge.  And also not have speedbooster.  But then timing the shinespark would be very difficult.